TransformDragger.rbxmx

```diff
@@ -6624,5 +6616,88 @@ return module
 ]]></ProtectedString>
 			</Properties>
 		</Item>
+		<Item class="ModuleScript" referent="RBXEC7C6085619E450693F32FA161266FC2">
+			<Properties>
+				<Content name="LinkedSource"><null></null></Content>
+				<string name="Name">RecyclingBin</string>
+				<string name="ScriptGuid"></string>
+				<ProtectedString name="Source"><![CDATA[-- @CloneTrooper1019 <3
+
+local recyclingBin = {}
+local bin = {}
+local expirationTime = 5
+
+function recyclingBin:Recycle(object)
+	local class = object.ClassName
+	if not bin[class] then
+		bin[class] = {}
+	end
+	bin[class][object] = tick()+expirationTime;
```

Comment:
It might be better to pass this in as parameter.

Ignore this: I hadn't seen the new changes.